README.md

```diff
@@ -87,6 +87,10 @@ Rollbar::init($config, $set_exception_handler, $set_error_handler);
 ?>
 ```
 
+### For CodeIgniter Users
+
+If you are using CodeIgniter and wish to report on E_NOTICE level errors you'll have to be sure to run `Rollbar::init` from inside the Controller constructor.
```

Comment:
Why is this? Does this mean that `Rollbar::init` will be called on every request? That seems like it's probably papering over something else that's not working as designed...